### ISSUE TITLE
Current stock view buttons not on screen to select 

### ISSUE THREAD

**Issue by dbruynb** [PantryParty/pantry_party#113]
Thanks for the much awaited update to support grocy ver 3. On iPhone Xr the buttons is not available in the stock view. Workaround at this time is the trun the phone side ways for the buttons to show correctly. 

See screen shots attached. 

![A746ABF0-C76C-492B-BCC9-9B7BD6229D93](https://user-images.githubusercontent.com/11042487/112020338-eae15f00-8b38-11eb-937c-7dd3836ce7a5.png)
![B1E5EC09-58E4-4A1B-83E0-D1EF26147151](https://user-images.githubusercontent.com/11042487/112020354-ef0d7c80-8b38-11eb-97ec-d73748abd78e.png)

**Comment by arjes** [comment]
Uh oh thanks for the report. I'll try to reproduce in the simulator.

Are you able to scan? I'm having trouble with a real device which I can't reproduce easily on the simulator. I'm trying to get that fix out asap.

**Comment by dbruynb** [comment]
Just tested now and scan doesn't work. I run IOS 14.5 public beta. The app crash and close when trying to scan. Did confirm that app has camera access in IOS settings. 

Let me know if I can test any thing else. I'm happy to assist.

**Comment by arjes** [comment]
Thanks it look like a lib compatibility problem. I think I have the problem
narrowed down.

**Comment by arjes** [comment]
A new release has been submitted for apple to review. It can take a few hours to a day to be approved and made available for distribution.  The camera worked in the simulator, so lets hope it works when it gets out in the wild

**Comment by arjes** [comment]
Hey 1.2.2 was just approved and should be available for download. Can you give the camera a try?

**Comment by dbruynb** [comment]
> Hey 1.2.2 was just approved and should be available for download. Can you give the camera a try?

Thanks, camera is working without any crashes. However the buttons is still out of view as per previous pictures attached.

**Comment by bmalinconico** [comment]
@dbruynb Yep I didn't get to that one yet, thanks for the confirmation on the camera.
